test/specs/column/tests/06.iframe_input.js

```diff
@@ -0,0 +1,154 @@
+      expect(columns.length).toBe(expectedColumns.length);
+
+      columns.forEach((col, i) => {
+        if (i >= expectedColumns.length) return;
+        expect(col.name).toBe(expectedColumns[i].name, `missmatch for name=${col.name}`);
+      });
+    });
+
+    it('isInputIframe should return the proper values.', () => {
+      columns.forEach((col, i) => {
+        if (i >= expectedColumns.length) return;
+        expect(col.isInputIframe).toBe(expectedColumns[i].isInputIframe, `missmatch for name=${col.name}`);
+      });
+    });
+
+    it('inputIframeProps should return the proper values.', () => {
+      columns.forEach((col, i) => {
+        if (i >= expectedColumns.length) return;
+        const expectedCol = expectedColumns[i];
+
+        if (!expectedCol.inputIframeProps) {
+          expect(col.inputIframeProps).toBeFalsy(`missmatch inputIframeProps for name=${col.name}`);
+
+          return;
+        }
+
+        expect(col.inputIframeProps).toBeTruthy(`missmatch inputIframeProps name=${col.name}`);
+
+        // .urlPattern
+        expect(col.inputIframeProps.urlPattern).toEqual(expectedCol.inputIframeProps.urlPattern, `urlPattern missmatch for name=${col.name}`);
+
+
+        // .columns
+        const mappedCols = col.inputIframeProps.columns;
+        const expectedMappedColNames = expectedCol.inputIframeProps.columnNames;
+        expect(mappedCols.length).toBe(expectedMappedColNames.length, `missmatch columns length for name=${col.name}`);
+        mappedCols.forEach((mappedCol, j) => {
+          if (j >= expectedMappedColNames.length) return;
+          expect(mappedCol.name).toBe(expectedMappedColNames[j], `missmatch columnnames name=${col.name}, mapped col=${mappedCol.name}`);
+        });
+
+        // .fieldMapping
+        const fieldMapping = col.inputIframeProps.fieldMapping;
+        const fieldNames = Object.keys(fieldMapping);
+
+        const expectedFieldMapping = expectedCol.inputIframeProps.fieldMapping;
+        const expectedFieldNames = Object.keys(expectedFieldMapping);
+
+        expect(fieldNames.length).toBe(expectedFieldNames.length, `missmatch fieldMapping length for name=${col.name}`);
+        fieldNames.forEach((fieldName, j) => {
+          if (j >= expectedFieldNames.length) return;
+          expect(fieldName).toBe(expectedFieldNames[j], `missmatch fieldMapping for name=${col.name}, fieldname=${fieldName.name}`);
+
+          expect(fieldMapping[fieldName].name).toBe(expectedFieldMapping[expectedFieldNames[j]], `missmatch fieldMapping for name=${col.name}, mapped fieldname=${fieldName.name}`);
+        });
+
+        // .optionalFieldNames
+        expect(col.inputIframeProps.optionalFieldNames).toEqual(expectedCol.inputIframeProps.optionalFieldNames, `optionalFieldNames missmatch for i=${i}, name=${col.name}`);
+
+      });
+    });
+
+
+    // TODO
+    describe('renderInputIframeUrl', () => {
```

Comment:
A few other TODOs I wasn't sure if you meant to finalize.

Reply:
Agh.. I completely forgot about these test cases. Thanks for catching it, I'll add them in my next commit.